lib/ibm_watson/base_service.rb

```diff
@@ -14,9 +14,10 @@ def version(version)
       end
     end
 
-    def initialize(username:, password:)
+    def initialize(username:, password:, timeouts: [:global, {write: 2, connect: 5, read: 5}])
```

Comment:
My only critique is the amazing obscureness of the format of this timeout parameter .. 
Can we use a proper structured object for this?